### ISSUE TITLE
led Multiplexer question

### ISSUE THREAD

**Issue by T4lus** [tttapa/Control-Surface#276]
Hi, Can I use multiplexer with led ?

**Comment by tttapa** [comment]
No, a multiplexer only has a single active output at any given time. You can use an LED driver with a shift register input.
